components/raftstore/src/store/fsm/store.rs

```diff
@@ -121,8 +122,11 @@ pub const ENTRY_CACHE_EVICT_TICK_DURATION: Duration = Duration::from_secs(1);
 pub const MULTI_FILES_SNAPSHOT_FEATURE: Feature = Feature::require(6, 1, 0); // it only makes sense for large region
 
 // Every 30 minutes, check if we can run full compaction. This allows the config
-// setting `periodic_full_compact_start_max_cpu` to be changed dynamically.
+// setting `periodic_full_compact_start_times` to be changed dynamically.
 const PERIODIC_FULL_COMPACT_TICK_INTERVAL_DURATION: Duration = Duration::from_secs(30 * 60);
+// If periodic full compaction is enabled (`periodic_full_compact_start_times`
+// is set), sample load metrics every 30 seconds.
+const LOAD_METRICS_WINDOW_DURATION: Duration = Duration::from_secs(30);
```

Comment:
how about calling it `STATS` instead of `METRICS`

Reply:
Done.